utils/config.go

```diff
@@ -17,6 +17,13 @@ const (
 	configFilePerm = 0644
 )
 
+const (
```

Comment:
I'm guessing this will be updated in a later PR?

Reply:
what needs to be updated?

Reply:
Sorry, I meant the names of the GPU binaries/libs.

Reply:
oh yeah we can change stuff around once we start pulling from cloudsmith